plugins/Login/PasswordResetter.php

```diff
@@ -383,7 +395,11 @@ protected function getUserInformation($loginOrMail)
      */
     protected function checkPasswordHash($passwordHash)
     {
-        UsersManager::checkPasswordHash($passwordHash, Piwik::translate('Login_ExceptionPasswordMD5HashExpected'));
+        $hashInfo = $this->passwordHelper->info($passwordHash);
+
+        if (!isset($hashInfo['algo']) || 0 >= $hashInfo['algo']) {
+            throw new Exception(Piwik::translate('Login_ExceptionPasswordMD5HashExpected'));
```

Comment:
Screenshots now look fine. Well, at least the ones I get listed (only those with differences?). The "all screenshots" button throws an exception at me and the tool's repository is private so I cannot get the real path (guest privileges) 🙄